README.md

```diff
@@ -12,6 +12,39 @@ source venv/bin/activate
 python3 -m pip install git+https://github.com/dataforgoodfr/12_taxobservatory.git
 ```
 
+## Running the PDF downloader
+
+To run the report downloader from the command line, you can invoke the `pdf_downloader` module:
+
+```
+python3 -m collecte.pdf_downloader company_names.csv googleapi_credentials.yaml
```

Comment:
* Please can you add `googleapi_credentials.yaml` to the `.gitignore` and potentially a `googleapi_credentials.yaml.example` to the repo (as for the .env file)? 
* Should we leave a company_names.csv file in the repo so that newcomers can run the download without too much configuration?